scripts/run_device_container.sh

```diff
@@ -8,6 +8,43 @@
 docker pull huntsmanarray/device_startup:latest
 
 #Run the docker container
+
 #Note that network=host to map container's ports to the hosts directly
-#This is OK for the pis etc. but probably not for the control computer
-docker run huntsmanarray/device_startup:latest --network host -it
+#This is OK for the pis etc. but probably not for the control computer.
+
+#-it allows the container to run in interactive mode.
+
+#--cap-add SYS_ADMIN --device /dev/fuse is necessary for the SSHFS mount.
+
+#-v ~/.ssh:/root/.ssh allows the container to use the credentials of the host
```

Comment:
too much commented out stuff here. If these are notes to self for control computer docker work, maybe put the comments into that new file?